frontend/src/pages/HistoryPage.vue

```diff
@@ -293,16 +88,15 @@ export default {
         },
         generateMapFeatures: function () {
             let features = [];
-            this.visitedPlaces.forEach(function (place, i) {
-
+            this.places.allIds.forEach((id, i) => {
                 let feature = {
                     'type' : 'Feature',
                     'geometry' : {
                         'type' : 'Point',
-                        'coordinates' : [place.longitude, place.latitude]
+                        'coordinates' : [this.places.byId[id].longitude, this.places.byId[id].latitude]
```

Comment:
Create a variable `place` and set it with `this.places.byId[id]`, it will better handle if place by id won't be found

Reply:
You can actually create some getter